artemis/src/test/java/tech/pegasys/artemis/cli/deposit/DepositRegisterCommandTest.java

```diff
@@ -57,7 +57,7 @@
   private static final Cipher CIPHER = new Cipher(AES_IV_PARAM);
   private static final KdfParam KDF_PARAM = new SCryptParam(32, 262144, 1, 8, SALT);
   private static final KeyStoreData VALIDATOR_KEYSTORE =
-      KeyStore.encrypt(BLS_PRIVATE_KEY, PASSWORD, "", KDF_PARAM, CIPHER);
+      KeyStore.encrypt(BLS_PRIVATE_KEY, Bytes32.random(), PASSWORD, "", KDF_PARAM, CIPHER);
```

Comment:
Can you use a constant public key value instead of the random value

Reply:
done